components/operator-service/src/main/java/com/wso2telco/dep/operatorservice/model/BlacklistWhitelistCountResponseDTO.java

```diff
@@ -0,0 +1,14 @@
+package com.wso2telco.dep.operatorservice.model;
+
+public class BlacklistWhitelistCountResponseDTO {
+
+    private int count;
```

Comment:
Will avoid using a DTO to hold a single value. Use the integer in all the places instead